### PR TITLE
Add info about how Intelligent Document Service works

### PR DESCRIPTION
**Pull request by Luyao-Zhang-1**

### CODE REVIEW

File: content/en/docs/appstore/app-services/intelligent-document-service.md [hunk 1]
```diff
@@ -10,18 +10,37 @@ tags: ["Document Service", "AI", "ML", "OCR", "Industrial", "Manufacturing"]
 
 The [Intelligent Document Service](https://marketplace.mendix.com/link/component/118388) app service uses AI and OCR technologies to extract text and key value pairs from images of documents in bulk. For more information, see the [Technical Provider](#technical-provider) section.
 
+### 1.1 How Intelligent Document Service Works
```
Comment (shaileshpuranikmx) [inline content/en/docs/appstore/app-services/intelligent-document-service.md]: Can we add a question mark at the end?  
Adding a below image at the start may be a good idea. Also, the overall process description stating 3 stages 
1. Training
2. Publish
3. Extraction
image

If you need to capture the high-quality image at your end you can browse to 
https://datacapture-appservices.mendixcloud.com/login.html

File: content/en/docs/appstore/app-services/intelligent-document-service.md [hunk 2]
```diff
@@ -10,18 +10,37 @@ tags: ["Document Service", "AI", "ML", "OCR", "Industrial", "Manufacturing"]
...
 
-### 1.1 Features
+#### 1.1.1 Training
```
Comment (shaileshpuranikmx) [inline content/en/docs/appstore/app-services/intelligent-document-service.md]: We can add a snapshot from the shared slides.
image

File: content/en/docs/appstore/app-services/intelligent-document-service.md [hunk 3]
```diff
@@ -10,18 +10,37 @@ tags: ["Document Service", "AI", "ML", "OCR", "Industrial", "Manufacturing"]
...
+
```
Comment (shaileshpuranikmx) [inline content/en/docs/appstore/app-services/intelligent-document-service.md:16]: Can we explain the training process? 
1. Uploading samples
2. For each sample identify the important/interested fields
3. Publish the model


image

File: content/en/docs/appstore/app-services/intelligent-document-service.md (hunk 1 above)
Comment (Luyao-Zhang-1) [reply content/en/docs/appstore/app-services/intelligent-document-service.md]: Thanks. I will implement these changes.

**Review by shaileshpuranikmx — APPROVED** [review]

**Pull request merged**